### DISCUSSION TITLE
guide to install on Ubuntu?

### DISCUSSION THREAD

**Discussion by alxsimon** [pop-os/cosmic-epoch#1255] [Q&A]
Hi, is there any guide to install on Ubuntu? I've tried cosmic epoch on Fedora at some point but I am now stuck on Ubuntu because of work. I would still like to try out cosmic on this machine.

I've seen this from last year https://github.com/pop-os/cosmic-epoch/issues/80, is this still a problem?

Thanks!
Alexis

**Comment by jordan-bravo** [comment]
I would also like to know how to install Cosmic on Ubuntu.

**Reply by andrewbanchich** [reply]
@jackpot51 is ubuntu / debian not supported? [the website](https://system76.com/cosmic/) says "Try it out on your favorite distro!" but oddly doesn't list ubuntu / debian.

**Reply by mmstick** [reply]
Ubuntu has to package it, same as other distributions have done.

**Comment by nicojank** [comment]
Perhaps you could compile it? if you have the know-how, you might also be able to  make a PPA. However, i don't know these things that well, so i wouldn't be able to help.
